sources/install/package_mobile.sh

```diff
@@ -67,6 +67,17 @@ function install_androguard() {
     add-to-list "androguard,https://github.com/androguard/androguard,Reverse engineering and analysis of Android applications"
 }
 
+function install_mobsf(){
+    colorecho "Installing Mobile Security Framework"
+    fapt wkhtmltopdf
+    git clone https://github.com/MobSF/Mobile-Security-Framework-MobSF /opt/tools/Mobile-Security-Framework-MobSF
+    cd /opt/tools/Mobile-Security-Framework-MobSF && ./setup.sh
+
+    add-aliases mobsf
+    add-history mobsf
```

Comment:
Add a test command : `add-test-command "mobsf ...."`

Reply:
That's not really offered by mobsf. Would that work as workaround: `poetry --directory /opt/tools/Mobile-Security-Framework-MobSF run python -c 'from mobsf.MobSF.settings import VERSION; print(VERSION)'`